packages/rrweb/src/record/observers/canvas/canvas-manager.ts

```diff
@@ -343,6 +343,9 @@ export class CanvasManager implements CanvasManagerInterface {
       getCanvas(canvasElement).forEach((canvas: HTMLCanvasElement) => {
         const id = this.mirror.getId(canvas);
         if (snapshotInProgressMap.get(id)) return;
+        // Don't do anything if canvas height/width is 0, otherwise causes
+        // `createImageBitmap()` to throw
+        if (canvas.width === 0 || canvas.height === 0) return;
```

Comment:
```suggestion
        if (!canvas.width || !canvas.height) return;
```

micro-size-optimization :D